vowpalwabbit/accumulate.cc

```diff
@@ -19,18 +19,21 @@ using namespace std;
 
```

Comment:
How are sparse models represented?

Alekh

On Tue, Jun 28, 2016 at 1:49 PM, John <notifications@github.com> wrote:

> In vowpalwabbit/accumulate.cc
> <https://github.com/JohnLangford/vowpal_wabbit/pull/1009#discussion_r68807169>
> :
>
> > @@ -19,18 +19,21 @@ using namespace std;
> >
>
> @alekh <https://github.com/alekh> accumulate isn't factored right to
> support sparse models. Can you think about how to do it right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/JohnLangford/vowpal_wabbit/pull/1009/files/d792eb33cc19d5e7b7214f45686afcb37a31ad45#r68807169>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAyIGeYAfKJiqdpLc-xo7YeAgaUpuVV3ks5qQV6_gaJpZM4Iwg44>
> .
>